### ISSUE TITLE
Support for inclusions

### ISSUE THREAD

**Issue by Hobart2967** [rliebz/tusk#64]
**Is your feature request related to a problem? Please describe.**
Because complex script collections can cause very long tusk.ymls, i'd like to split up those tasks into certain files

**Describe the solution you'd like**
```
tasks:
  build: 
    include: .tusk/build.yml
  test:
    run: 
      - command: yarn test
```

**Comment by rliebz** [comment]
Thanks for opening this issue, definitely something I'm interested in.

This would have some challenges in terms of variable life-cycles. My intuition is that a first pass might have pretty limited support for referring to global options or other tasks, although I might consider limiting cross-file references a benefit to readability rather than a limitation.

The other potential syntax for this kind of behavior I've considered before would look like this:
```yaml
include:
  - .tusk/build.yml

tasks:
  test:
    run:
      - command: yarn test
```

With the advantage that each included file could be a full proper `tusk.yml`, and the downside that it opens up questions of how to resolve global options or tasks that share names across files.

**Comment by rliebz** [comment]
Now available in v0.5.2

Docs: https://rliebz.github.io/tusk/spec/#include
